themes/classic/templates/_partials/header.tpl

```diff
@@ -54,22 +54,32 @@
   </nav>
 {/block}
 
+{function renderLogo}
+  <a href="{$urls.pages.index}">
+    <img
+      class="logo img-fluid"
+      src="{$shop.logo.src}"
+      alt="{$shop.name}"
+      loading="lazy"
+      width="{$shop.logo.width}"
+      height="{$shop.logo.height}">
+  </a>
+{/function}
+
 {block name='header_top'}
   <div class="header-top">
     <div class="container">
        <div class="row">
         <div class="col-md-2 hidden-sm-down" id="_desktop_logo">
+          {if $shop.logo}
             {if $page.page_name == 'index'}
               <h1>
-                <a href="{$urls.pages.index}">
-                  <img class="logo img-responsive" src="{$shop.logo}" alt="{$shop.name}" loading="lazy" width="100" height="28">
-                </a>
+                {renderLogo}
```

Comment:
This thing looks so weird to me (not related to the PR), I think we should add a title somewhere else that surrounding the logo, right?

Reply:
@NeOMakinG yea it is weird it was introduced in https://github.com/PrestaShop/PrestaShop/commit/57269252d42c26b6e50974db6a7890ece837dd95#diff-8ada98d9b9b82420991362832e15678cdd981b2eaef44cc880074a98271dbb9c 
I am not SEO expert so I have no idea why it is here.

Reply:
Usually in my past jobs, we were adding an h1 inside the topbar, so the h1 is always a text on top level, and it was working pretty fine respecting SEO standards, I think we need to change this in the future (maybe in the new theme?)